File: trail1/장학금/scholarship.cpp
#include <iostream>
using namespace std;

int main() {
    int A, B;
    cin >> A >> B;
    int score;
    if (A >= 90) {
        if (B >= 95) {score = 100000;}
        else if (B >= 90) {score = 50000;}
        else {score = 0;}
    }
    else {score = 0;}
    cout << score << endl;
    return 0;
}

File: trail1/장학금/scholarship.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int a = sc.nextInt();
        int b = sc.nextInt();
        int score = 0;
        if (a >= 90 && b >= 95) {score = 100000;}
        else if (a >= 90 && b >= 90) {score = 50000;}
        System.out.println(score); 
    }
}